### ISSUE TITLE
What is the state of SchildiChat development? Last commit ~7 months ago.

### ISSUE THREAD

**Issue by dreirund** [SchildiChat/schildichat-desktop#261]
Ahoj,

I see that the last commit to this repository was ~7 months ago.

Does anyone know what the state of this software is? Is it still maintained (maintainers just doing a break)?, is it getting abandoned?

Regards!

**Comment by SpiritCroc** [comment]
We're not planning to abandon it, but we cannot predict when we'll have enough time to properly rebase on top of latest element. The current state of code is no longer reasonably maintainable in a way that keeps up with upstream changes, since Element changed too many things around, so we basically need to redo it more or less from scratch. You can check out the `lite` branch to see what's going on for the rebase, but it's barely something you could call "SchildiChat" at this point, since it's missing many of our changes (though we will either way drop some of them to get running again).

**Comment by dreirund** [comment]
> *We're not planning to abandon it, but we cannot predict when we'll have enough time to properly rebase on top of latest element.*

OK!

So feel free to close this issue since the question seems to be solved and it does not really seem to be an "issue".

But:

1. If you think this should stay opened (and pinned?) so that users know, feel free to leave it open.
2. I think it is helpful if you add a prominent message to the [`README.md`](https://github.com/SchildiChat/schildichat-desktop/blob/sc/README.md) stating what the current state is (i.e. out of date, and the [`lite` branch](https://github.com/SchildiChat/schildichat-desktop/tree/lite) beeing up to date but only with minimal changes from Element).

Regards!

**Comment by Ryonez** [comment]
I hadn't realized things were trailing so far currently.

I've only recently got Authenticated media sorted, Element and Schildichat android default to using this and fail to load without it set up. Part of my work involved my testing forcing authenticatied media endpoints only, and I was surprised when Schildichat web failed.
Looking at things, it was added to Element-Web in this version: https://github.com/element-hq/element-web/releases/tag/v1.11.70
It's under `Use stable endpoints for [MSC3916]` So took me a little to find.

The old `/_matrix/media` endpoint has been marked as deprecated, it's only a matter of time until Schildichat web will be unusable for images without an update.

Though I'll admit I'd be daunted with trying to work on rebasing myself, they've changed a lot...
